[cli] Add windows service commands (#26442)
hashicorp/nomad: website/content/commands/windows/service-uninstall.mdx

---
layout: docs
page_title: 'nomad windows service uninstall command reference'
description: |
  The `nomad windows service uninstall` command removes the nomad
  Windows service.
---

# `nomad windows service uninstall` command reference

The `windows service uninstall` command removes the nomad Windows service.

## Usage

```plaintext
nomad windows service uninstall
```

> The `windows service uninstall` command will stop the nomad service if
> it is currently running, deregister the service with the Windows eventlog,
> and remove the Windows service. The installed nomad binary, as well as
> the data and configuration directories, will not be removed.

Reviewer: ```suggestion
The `windows service uninstall` command stops the Nomad service if
it is currently running, deregisters the service with the Windows eventlog,
and removes the Windows service. The installed Nomad binary, as well as
the data and configuration directories, are not removed.
```
